frontend/src/components/pages/CreateReview/CreateReviewPage.tsx

```diff
@@ -10,12 +10,13 @@ import {
   RadioGroup,
   Stack,
   Text,
+  useDisclosure,
 } from "@chakra-ui/react";
```

Comment:
I get the following issues on the console when I first load the createReviewPage component:
![image](https://user-images.githubusercontent.com/57159970/157335992-14309ecf-6abd-4dc9-ad4c-068aaac2193e.png)
![image](https://user-images.githubusercontent.com/57159970/157336017-4e283f08-6a55-4b51-90ae-d57cca91c928.png)
![image](https://user-images.githubusercontent.com/57159970/157336041-1f6a8668-3b95-47a9-9d12-ca26063ef4ce.png)

I couldn't find out what was causing this issue. It might be a problem with my local code but I thought I would bring it up in case it is consistent for other ppl too

Reply:
Yea i see it as well hmmmm i'm not too sure what's causing it. Seems like it's a bug on ChakraUI's side that they've fixed with a [recent update](https://github.com/chakra-ui/chakra-ui/issues/4891)?